dir.targets

```diff
@@ -63,6 +63,7 @@
     <CommonTestProjectLockJson>$(MSBuildThisFileDirectory)src/Common/test-runtime/project.lock.json</CommonTestProjectLockJson>
     <CommonOutputTestProjectJson>$(GeneratedProjectJsonDir)/project.json</CommonOutputTestProjectJson>
     <UsesCommonTestProjectJson Condition="'$(IsTestProject)' == 'true' and '$(ProjectJson)'=='' ">true</UsesCommonTestProjectJson>
+    <SkipGenerateTestProjectJson Condition="'$(UsesCommonTestProjectJson)'==''">false</SkipGenerateTestProjectJson>
```

Comment:
This entire property set should be moved above the import of build.common.targets